obj/tasks.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
+    //DOM elements
+    createTaskBtn : '[type="button"]',
+    saveTaskBtn : '[type="submit"]',
+    nameField : '#taskDetailsForm\\.name',
+    descriptionField : '#taskDetailsForm\\.description',
+    billDropdown : '.Select-multi-value-wrapper',
+    billYesOption : '[aria-label="Yes"]',
+    rateField : "[name='taskDetailsForm.rate']",
+    nameSearchField : ".ag-header-cell:nth-of-type(1) .field__text--small",
```

Comment:
as far as i know, it can't. Since there are two indentical search fields i have to use nth-of-type. Alternatively i could leave only ".field__text--small", this would still work, but i would get a warning that selector returns 2 elements and that the first one will be selected.